packages/node/src/indexer/dynamic-ds.service.ts

```diff
@@ -12,9 +12,9 @@ import { MetadataRepo } from './entities/Metadata.entity';
 
 const logger = getLogger('dynamic-ds');
 
-const METADATA_KEY = 'dynamicDatasources';
+const METADATA_KEY = 'dynamicDataSources';
```

Comment:
Why is the casing here changed? Can all this please be reverted along with the rest of the changes in this file.

We would like minimal changes and make it easier to review, and reduce the chances of introducing bugs.